css/leaderboard-style.css

```diff
@@ -0,0 +1,138 @@
+body{
+    
+    background:linear-gradient(rgba(0,0,0,0.6), rgba(0,0,0,0.8)),url("../images/leaderboard.jpg");
+    background-size:cover;
+    background-position: center;
+}
+
+.section-leaderboard{
+    height:100vh;
+    
+    width:100%;
+    
+    padding:50px 0;
+    font-size:40px;
+    font-family: 'Lato';
+    text-align: center;
+  
+    
+
+}
+
+.section-leaderboard .section-title{
+    text-transform: capitalize;
+    color:white;
+    font-weight: 500;
+}
+
+.section-leaderboard .container{
+   
+    margin:30px auto 0 auto!important;
+    border-radius:5px;
+    padding:3px;
+    background-color:#FEFEFE;
+    width:60%;
+    
+}
+
+.section-leaderboard table{
+    width:100%;
+}
+
+.section-leaderboard table tr:first-of-type{
+    background-color:black;
+    color:white;
+    
+    font-weight:normal;
+    
+    text-transform:uppercase;
+    border-bottom:3px solid white;
+    
+    
+}
+
+.section-leaderboard table tr:first-of-type th{
+    text-align: center;
+    padding:10px 0;
+    font-family: 'Lato', sans-serif!important;
+    font-size:100%;
+    font-weight: 300;
+    
+}
+
+.section-leaderboard table tr{
+    font-size:46%;
+    font-weight:300;
+    padding:5px 0;
+    margin:3px 0;
+    background-color:white;
+    
+   
+}
+
+.section-leaderboard table td{
+   
+    padding:5px 0;
+   
+}
+
+.section-leaderboard table tr:nth-of-type(2),
+.section-leaderboard table tr:nth-of-type(3),
+.section-leaderboard table tr:nth-of-type(4),
+.section-leaderboard table tr:nth-of-type(5),
+.section-leaderboard table tr:nth-of-type(6){
```

Comment:
will it work only for 6 rows?

Reply:
the colour change to green is only for first 5,the rest wil be in white